src/worker/storage.ts

```diff
@@ -140,15 +140,14 @@ export default class Storage {
    */
   updateProjectData(metadata: { [key: string]: string }): Promise<void> {
     return this.db.transaction("readwrite", this.db.projectData, async () => {
-      await this.db.projectData.put({
-        langName: metadata.langName,
-        bcp47Tag: metadata.bcp47Tag,
-        authorName: metadata.authorName,
-        modelID: metadata.modelID,
-        copyright: metadata.copyright,
-        version: metadata.version,
-        id: PACKAGE_ID,
-      });
+      const existingMetadata:
+        | StoredProjectData
```

Comment:
Out of curiosity, what do you call the `|` symbol in this context? I can only find references to it as a bitwise OR operator

Reply:
I'm not sure if it has a name in this context. According to the TypeScript docs:

> We use the vertical bar (`|`) to separate each type, so `number | string | boolean` is the type of a value that can be a `number`, a `string`, or a `boolean`.
>
> _Source_: https://www.typescriptlang.org/docs/handbook/unions-and-intersections.html

Reply:
Type union operator? Or it separates elements of a type union? A sum type operator? I don't really know, haha!